llvm/tools/llvm-objcopy/ObjcopyOptions.cpp

```diff
@@ -584,6 +584,69 @@ static Expected<int64_t> parseChangeSectionLMA(StringRef ArgValue,
   return *LMAValue;
 }
 
+static Expected<SectionPatternAddressUpdate>
+parseChangeSectionAddr(StringRef ArgValue, StringRef OptionName,
+                       MatchStyle SectionMatchStyle,
+                       function_ref<Error(Error)> ErrorCallback) {
+
+  SectionPatternAddressUpdate PatternUpdate;
+
+  size_t last_i = ArgValue.find_last_of("+-=");
```

Comment:
Name doesn't conform to [coding standards](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly).